torch/onnx/symbolic_opset11.py

```diff
@@ -356,3 +356,25 @@ def index_copy(g, self, dim, index, source):
         return g.op("ATen", self, index, source, dim_i=dim_value, operator_s="index_copy")
     expanded_index_shape, expanded_index = sym_help._index_fill_reshape_helper(g, self, dim, index)
     return scatter(g, self, dim, expanded_index, source)
+
+
+def __rshift_(g, self, other):
+    if self.type().scalarType() == 'Byte':
+        if other.type().scalarType != 'Byte':
+            other = g.op("Cast", other, to_i=sym_help.cast_pytorch_to_onnx['Byte'])
+        return g.op('BitShift', self, other, direction_s="RIGHT")
+    two = g.op('Constant', value_t=torch.tensor(2, dtype=torch.float))
+    two_pow = g.op('Pow', two, other)
```

Comment:
There is probably an edge case not coverred. When `self` is integral, and `other` is floating point, pytorch will cast `other` to integral as well. 

Please try
```python
        class BitshiftModel(torch.nn.Module):
            def forward(self, input):
                return input << 2.4
        input = torch.arange(24, dtype=torch.long).reshape(3, 4, 2)
        self.run_test(BitshiftModel(), input)
```

Reply:
good catch. I updated the code to take this into account.